flutter_local_notifications/lib/src/platform_specifics/darwin/initialization_settings.dart

```diff
@@ -51,6 +52,19 @@ class DarwinInitializationSettings {
   /// Default value is 'false'.
   final bool requestCriticalPermission;
 
+  /// Request permission to provide custom notification settings UI.
+  ///
+  /// When enabled, the system displays a "Configure Notifications in {App}"
+  /// button in the notification's context menu (when swiping on a
+  /// notification and selecting Options). This allows your app to
+  /// present a custom UI for notification settings by handling the
+  /// appropriate delegate callback.
+  ///
+  /// Corresponds to https://developer.apple.com/documentation/usernotifications/unauthorizationoptions/3040321-providesappnotificationsettings
+  ///
+  /// Default value is false.
```

Comment:
```suggestion
  /// Corresponds to https://developer.apple.com/documentation/usernotifications/unauthorizationoptions/3040321-providesappnotificationsettings
  ///
  /// On iOS, this property is only applicable to iOS 12 or newer.
  /// On macOS, this property is only applicable to macOS 10.14 or newer.
  ///
  /// Default value is false.
```

Reply:
For consistency can you make the paragraph with the applicable OS versions last?

Reply:
Done (edited pending suggestion above)